frontend/app/helpers/spree/store_helper.rb

```diff
@@ -0,0 +1,35 @@
+module Spree
+  module StoreHelper
+    def store_country_iso(store)
+      store ||= current_store
+      return unless store
+      return unless store.default_country
```

Comment:
Maybe would be better to rewrite this 2 line like this
```suggestion
      return unless store&.default_country
```